packages/python/plotly/plotly/figure_factory/_gantt.py

```diff
@@ -190,6 +260,7 @@ def gantt_colorscale(
     task_names=None,
     data=None,
     group_tasks=False,
+    show_hover_fill=True,
```

Comment:
When tasks are packed tightly (even with overlap) it can be hard to get the hoverinfo for a given trace, if there is only one marker for that. I think, having the option to see info on the fill could be helpful (but maybe this is not that important, since traces can be turned off now :)